src/main/java/com/google/graphgeckos/dashboard/fetchers/github/GitHubFetcher.java

```diff
@@ -0,0 +1,25 @@
+package com.google.graphgeckos.dashboard.fetchers.github;
+
+import com.google.graphgeckos.dashboard.fetchers.github.CommitData;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GitHubFetcher {
+
+  /**
+   * Handles POST requests from the GitHub API. Setting GitHub Webhooks is required.
+   * Extracts required data from the json {@see CommitData} and adds it to the Storage.
+   *
+   * Class, that provide access to the Storage, is required to implement the functionality.
+   * @param commitData json from the GitHub API.
+   */
+  @PostMapping("/github-info")
+  public void postGitHubInfo(@RequestBody CommitData commitData) {
+    System.out.println(commitData);
+    // TODO: implement described functionality. (Couldn't be implemented now since there is no Class to access storage).
```

Comment:
Todos become more useful if you add a reference link to a GitHub issue to them or and issue number.

TODO(link): []

This way, people reading the code can look up the issue and ask questions related to the TODO or see the current state / prioritization of it.

Reply:
Done!